linkerd/app/inbound/src/metrics/authz.rs

```diff
@@ -52,6 +57,13 @@ struct TcpInner {
     terminate: Mutex<HashMap<ServerKey, Counter>>,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct HTTPLocalRateLimitLabels {
+    pub server: ServerLabel,
+    pub rate_limit: Option<Arc<Meta>>,
+    pub scope: String,
```

Comment:
If we update this to reflect that scopes are known statically, we can avoid needless string allocation

```suggestion
    pub scope: &'static str,
```